web/assets/images/uit-logo.svg

```diff
@@ -1,6 +1,49 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" id="Layer_1" x="0px" y="0px" width="40px" height="23px" viewBox="0 0 40 23" enable-background="new 0 0 40 23" xml:space="preserve">
-<path fill="#1D1D1B" d="M13.337,13.574c0,1.563-0.166,2.822-1.368,3.947c-1.074,1.005-2.387,1.355-3.7,1.469  c-1.146,0.1-2.643-0.073-3.607-0.75c-1.331-0.957-1.587-2.372-1.587-4.178V4.104l3.902,0.324v9.024c0,0.484-0.019,1.138,0.236,1.572  c0.292,0.449,0.875,0.61,1.366,0.589c0.439-0.021,0.985-0.188,1.313-0.53c0.493-0.487,0.438-1.213,0.438-1.862V4.706l3.007,0.25  V13.574z"/>
-<path fill="#1D1D1B" d="M18.13,8.323l-3.609-0.159V5.055l3.609,0.301V8.323z M18.13,17.896l-3.609,0.297V9.187l3.609,0.112V17.896z"/>
-<polygon fill="#1D1D1B" points="25.329,17.363 21.684,17.666 21.684,8.542 19.059,8.427 19.059,5.433 27.898,6.167 27.898,8.81   25.329,8.699 "/>
-<polygon fill="#1D1D1B" points="0,0 38,0 38,23 0,23 30.399,19.168 30.399,3.834 "/>
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 25.2.3, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Sometimes it can be a good idea to tidy up the svg a bit up.
I use this tool:
https://jakearchibald.github.io/svgomg/

Reply:
Nice tip! I cleaned up the SVG with the tool.